src/std/mod.rs

```diff
@@ -399,6 +404,39 @@ where
     }
 }
 
+fn convert_fee_details(details: FeeDetails<NumberOrHex>) -> ApiResult<FeeDetails<u128>> {
+    let inclusion_fee = inclusion_fee_with_balance(details.inclusion_fee)?;
+    let tip = details
+        .tip
+        .try_into()
+        .map_err(|_| ApiClientError::TryFromIntError)?;
+    Ok(FeeDetails { inclusion_fee, tip })
+}
+
+fn inclusion_fee_with_balance(
+    inclusion_fee: Option<InclusionFee<NumberOrHex>>,
```

Comment:
I think an Option as a sole argument does not make much sense. I think it is better to check on the outside if it is set with a `map_or_default` or something like that. What do you think?

Reply:
Yeah, that's much better. Let me change it

Reply:
Changed to if-let so I can `?` the result

Reply:
Yest this is fine with me, I just think it is good to see on top-level that nothing happens with a `None`.

One thing I came to like recently (but this is only a matter of style) would be:

`fee_opt.map(inclusion_fee_with_balance).transpose()?`

I don't think it needs to be changed, just letting you know because I think this pattern is not well known. :)